shame_script.py

```diff
@@ -272,4 +273,23 @@ async def signup_passthrough(
         logger.exception("Error during signup")
 
 
+@discord.app_commands.describe(user_to_shame="Mention of user")
+@bot.tree.command(name="shame")
+async def shame_passthrough(
+    interaction: discord.Interaction, user_to_shame: discord.Member
+):
+    logger.info("Shame command received for user: %s", user_to_shame.name)
+
+    await interaction.response.defer(ephemeral=False, thinking=True)
+    try:
+        await shame(interaction, user_to_shame)
+        logger.info("Shaming successful for user: %s", user_to_shame.name)
+
+    except Exception as e:
+        logger.exception("Error during shaming: %s", e)
+        await interaction.followup.send(
+            "An error occurred while processing the shame command."
+        )
```

Comment:
very nice, should be added to all slash commands

Reply:
maybe a followup change to add a base 'Command'